collections-csharp/HashtableInCSharp/HashtableInCSharp/HashtableDemo.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+
+namespace HashtableInCSharp
+{
+    public class HashtableDemo
+    {
+        public Hashtable CreateHashTable()
+        {
+            var firstHashTable = new Hashtable();
+
+            var secondHashTable = new Hashtable(100);
+
+            var dictionary = new Dictionary<int, User>()
+            {
+                { 1, new User() { FirstName = "Sam", LastName = "Manalt"} },
+                { 2, new User() { FirstName = "Lone", LastName = "Costa"}},
+                { 3, new User() { FirstName = "Alberto", LastName = "Daci"} },
+            };
+            
+            var thirdHashTable = new Hashtable(dictionary);
+
+            return secondHashTable;
+        }
+
+        public Hashtable AddToHashTable(Hashtable hashTable)
```

Comment:
This, in my opinion, would be more intuitive as an extension method. I would also suggest renaming it to "AddSampleDataToHashTable"